cmd/drone-server/server.go

```diff
@@ -38,12 +38,12 @@ var flags = []cli.Flag{
 	cli.BoolFlag{
 		EnvVar: "DRONE_DEBUG",
 		Name:   "debug",
-		Usage:  "start the server in debug mode",
+		Usage:  "enable server debug mode",
 	},
 	cli.StringFlag{
 		EnvVar: "DRONE_SERVER_HOST,DRONE_HOST",
 		Name:   "server-host",
-		Usage:  "server host",
+		Usage:  "server host (<scheme>://<hostname>)",
```

Comment:
Maybe call it `server fqdn`? That should be more descriptive since it's not only the host name which led to conusion in the past.